corehq/apps/app_manager/views/formdesigner.py

```diff
@@ -102,13 +101,14 @@ def form_source_legacy(request, domain, app_id, module_id=None, form_id=None):
 
 
 def get_form_submit_history_url_for_last_30_days(request, domain, app, module, form):
+    from corehq.util.view_utils import reverse
```

Comment:
Did it cause circular import issues to have this at the top of the file, or is it just the name conflict with `django.urls.reverse`? If it's the latter, I would prefer to see it renamed during import, something like this, instead of the local import https://github.com/dimagi/commcare-hq/blob/c3a157e43b63a78fa4e323d3e27b764d73ec1723/corehq/apps/data_interfaces/views.py#L95